security/guard_authentication.rst

```diff
@@ -505,7 +505,7 @@ and add the following logic::
             $csrfToken = $request->request->get('_csrf_token');
             
             if (false === $this->csrfTokenManager->isTokenValid(new CsrfToken('authenticate', $csrfToken))) {
-                throw new InvalidCsrfTokenException('Invalid CSRF token.');
+                return null;
```

Comment:
Hello @rubenrubiob, as said in https://github.com/symfony/symfony-docs/pull/7996, such change is wrong as it would allow another guard to try to authenticate this request. To stop the process here an exception needs to be thrown.